clang/test/SemaCXX/cxx2c-delete-with-message.cpp

```diff
@@ -0,0 +1,30 @@
+// RUN: %clang_cc1 -std=c++2c -fsyntax-only -verify %s
```

Comment:
Can you add tests with unicode in the message?

Reply:
That didn’t even occur to me; thanks for pointing that out.

Reply:
Done.